.pre-commit-config.yaml

```diff
@@ -53,3 +53,12 @@ repos:
       - id: mypy
         exclude: ^(docs|tests)/.*$
         args: ["--ignore-missing-imports"]
+
+  - repo: https://github.com/pre-commit/mirrors-pylint
+    rev: v2.6.0
+    hooks:
+      - id: pylint
```

Comment:
What's the difference between using this and adding pylint as a command in `tox.ini`? Because this should be part of gating too.

Reply:
There is no difference it is redundant check.